.storybook/webpack.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  resolve: {
+    alias: {
+      "react-impl": "react"
```

Comment:
`preact` is used in examples components, and `preact-compat` is the substitution for React in libraries, such as storybook and FlipMove itself